### ISSUE TITLE
Angular 9 Prod Build Error with Terser. ERROR in polyfills-es5.js from Terser Invalid assignment.

### ISSUE THREAD

**Issue by amilaI** [just-jeb/angular-builders#691]
##  Bug Overview

When building the Angular 9 app via **ng build --prod**, I'm getting the following error, however, **ng serve** works fine.

`ERROR in polyfills-es5.js from Terser
Invalid assignment [polyfills-es5.js:9,19]`

    at ee (/Users/a/inspire/node_modules/terser/dist/bundle.min.js:1:19453)
    at c (/Users/a/inspire/node_modules/terser/dist/bundle.min.js:1:27932)
    at ze (/Users/a/inspire/node_modules/terser/dist/bundle.min.js:1:50127)
    at qe (/Users/a/inspire/node_modules/terser/dist/bundle.min.js:1:50208)
    at T (/Users/a/inspire/node_modules/terser/dist/bundle.min.js:1:33828)
    at /Users/a/inspire/node_modules/terser/dist/bundle.min.js:1:30197
    at /Users/a/inspire/node_modules/terser/dist/bundle.min.js:1:28664
    at /Users/a/inspire/node_modules/terser/dist/bundle.min.js:1:32055
    at /Users/a/inspire/node_modules/terser/dist/bundle.min.js:1:32118
    at /Users/a/inspire/node_modules/terser/dist/bundle.min.js:1:28664`

## Minimal Reproduction

Here's my pacakge.json.
```
 "dependencies": {
    "@angular-builders/custom-webpack": "^9.0.0",
    "@angular/animations": "~9.0.2",
    "@angular/cdk": "^9.1.0",
    "@angular/common": "~9.0.2",
    "@angular/compiler": "~9.0.2",
    "@angular/core": "~9.0.2",
    "@angular/flex-layout": "^9.0.0-beta.29",
    "@angular/forms": "~9.0.2",
    "@angular/material": "^9.1.0",
    "@angular/platform-browser": "~9.0.2",
    "@angular/platform-browser-dynamic": "~9.0.2",
    "@angular/router": "~9.0.2",
    "classlist.js": "^1.1.20150312",
    "jquery": "^3.4.1",
    "ngx-slick-carousel": "^0.4.7",
    "rxjs": "~6.5.4",
    "single-spa-angular": "^3.1.0",
    "slick-carousel": "^1.8.1",
    "tslib": "^1.11.0",
    "web-animations-js": "^2.3.2",
    "zone.js": "~0.10.2"
  },
  "devDependencies": {
    "@angular-devkit/build-angular": "~0.900.3",
    "@angular/cli": "~9.0.3",
    "@angular/compiler-cli": "~9.0.2",
    "@angular/language-service": "~9.0.2",
    "@types/jasmine": "~3.5.7",
    "@types/jasminewd2": "~2.0.8",
    "@types/node": "^13.7.6",
    "codelyzer": "^5.2.1",
    "jasmine-core": "~3.5.0",
    "jasmine-spec-reporter": "~4.2.1",
    "karma": "~4.4.1",
    "karma-chrome-launcher": "~3.1.0",
    "karma-coverage-istanbul-reporter": "~2.1.1",
    "karma-jasmine": "~3.1.1",
    "karma-jasmine-html-reporter": "^1.5.2",
    "protractor": "~5.4.3",
    "ts-node": "~8.6.2",
    "tslint": "~6.0.0",
    "typescript": "~3.7.4"
  }
```
Major versions

**Comment by just-jeb** [comment]
Gonna need more info. Could you provide a reproduction repo please?

**Comment by amilaI** [comment]
> Gonna need more info. Could you provide a reproduction repo, please?
Thanks for the quick prompt!

I spent quite a lot of time to resolve this issue but still failed! I am using https://github.com/single-spa/single-spa-angular and @angular-builders/custom-webpack to build a micro frontend in Angular. It worked perfectly fine with Angular 8. The issue comes when I update the project into Angular 9.

I was able to find out that code inside customWebpackConfig is causing the problem. It means this is something to do with single-spa-angular implementation. It seems single-spa-angular does not support Angular 9 at the moment.

Hence I suspect this issue is related to https://github.com/single-spa/single-spa-angular.
